File: Contacts/src/Contacts/src/main/java/com/example/conrad/contacts/MainActivity.java
package com.example.conrad.contacts;

import android.app.Activity;
import android.content.Intent;
import android.database.Cursor;
import android.net.Uri;
import android.provider.ContactsContract;
import android.support.v4.app.FragmentActivity;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.Toast;

import static android.R.attr.data;

public class MainActivity extends FragmentActivity {

    @Override
    protected void onCreate (Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

//    @Override
//    protected void onActivityResult (int requestCode, int resultCode,
//                                    Intent intent) {
//        Uri contactData = data.getData();
//        Cursor c = getContentResolver().query(contactData, null, null, null, null);
//        if (c.moveToFirst()) {
//            int phoneIndex = getColumnIndex(ContactsContract.CommonDataKinds.Phone.NUMBER);
//            String num = c.getString(phoneIndex);
//            Toast.makeText(MainActivity.this, "Number=" + num, Toast.LENGTH_LONG).show();
//        }
//    }

    public void contactList () {
        Intent pickContact = new Intent(Intent.ACTION_PICK, ContactsContract.Contacts.CONTENT_URI);
        pickContact.setType(ContactsContract.CommonDataKinds.Phone.CONTENT_TYPE);
        startActivityForResult(pickContact, 1);
    }

}

File: AdmonereTest/src/main/java/com/example/conrad/admoneretest/MainActivity.kt
package com.example.conrad.admoneretest

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Button

class MainActivity : AppCompatActivity() {

    private var curTab = 0

    fun setTab (tabs : Array<Button>) {
        for (tab in 0 until tabs.size) {
            if (curTab == tab) tabs[tab].setBackgroundResource(R.drawable.pagerectanglebuttonselected)
            else tabs[tab].setBackgroundResource(R.drawable.pagerectanglebutton)
        }
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val ongoingBtn = findViewById<View>(R.id.ongoingButton) as Button
        val pastBtn = findViewById<View>(R.id.pastButton) as Button

        val tabs = arrayOf(ongoingBtn, pastBtn)
        setTab(tabs)

        tabs.forEach { it.setOnClickListener {
            if (tabs.indexOf(it) != curTab) {
                curTab = tabs.indexOf(it)
                setTab(tabs)
            }
        } }



    }
}
